### DISCUSSION TITLE
Is it possible to add JS or CSS file to plugin?

### DISCUSSION THREAD

**Discussion by saman222** [joomengine/Joomla-Component-Builder#746] [Ideas]
Hi
i want to add CSS and JS file to plugin (or module). now i can not see any option to add file such as JavaScript & CSS Tab in the site view of component.
is it possible to implement that in your future version and i want is there any way to handle my need until you maybe want add this feature.
thank you

**Comment by Llewellynvdm** [comment]
Yes it is possible already, when you are looking at the list of your plugins in JCB, you will see there is a button to add files and folders. This is one way to include your own CSS, and JS. But you will manually need to add it to your plugin.

With modules it is easier, you can create a library and just add the library to your module... libraries can be CSS, JS and images. Here is a [little tutorial](https://www.youtube.com/watch?v=rDjvgLYOt1o&list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE&index=49) of how to create a library. Note that the library will be added to display area.

To add JS, or CSS to fields in the Admin area of a module, just add the JS or CSS to the field. You will see there is a tab called scripts, read about this [feature here more](https://github.com/vdm-io/Joomla-Component-Builder/issues/609#issuecomment-739134242).
